Jbarget/oxymore
# updates advertising copy and adds new image

### What changes have you made?
- added the third advertising image `honest-coupons` 
- replaced the placeholder copy with the approved copy for all adverts 

### Which issue(s) does this PR fix?

Fixes #277 

### Screenshots (if there are design changes)

<img width="268" alt="Screenshot 2020-12-08 at 14 07 29" src="https://user-images.githubusercontent.com/53219789/101487856-0a27f800-395f-11eb-814b-9b5c843a50fb.png">


### How to test

Go to `localhost:3000/advertising` and check that there are three images with three translatable texts
